src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const currentRent = 40;
+  const minDaysOff = 3;
+  const maxDaysOff = 7;
+  const minDiscountRent = 20;
+  const maxDiscountRent = 50;
```

Comment:
The constants defined here represent configuration for the rental cost calculation. It's good practice to use uppercase with underscores for such constants to indicate that they are not meant to change.